types/debug/index.d.ts

```diff
@@ -28,6 +30,7 @@ declare namespace debug {
 
     export interface IDebugger {
         (formatter: any, ...args: any[]): void;
+        (...args: string[]): void;
```

Comment:
I'm not so sure about this. The previous signature yields a legitimate error. You need at least one argument to ```debug```, that's the point of having the typing in the first place. With your overload, the following becomes OK, but it shouldn't:

```ts
var empty = []
log(...empty)
```